File: VB/T191577/Reports/CategoriesReport.vb
Imports System
Imports System.Drawing
Imports System.Collections
Imports System.ComponentModel
Imports DevExpress.XtraReports.UI

Namespace T191577.Reports
	Partial Public Class CategoriesReport
		Inherits DevExpress.XtraReports.UI.XtraReport

		Public Sub New()
			InitializeComponent()
			pbPicture.DataBindings.Add("Image", Nothing, "Picture")
		End Sub

		Private Sub CategoriesReport_DataSourceDemanded(ByVal sender As Object, ByVal e As EventArgs) Handles MyBase.DataSourceDemanded
			Me.DataSource = Category.GetCategories()
		End Sub

	End Class
End Namespace

File: CS/T191577/Reports/CategoriesReport.cs
using System;
using System.Drawing;
using System.Collections;
using System.ComponentModel;
using DevExpress.XtraReports.UI;

namespace T191577.Reports {
    public partial class CategoriesReport : DevExpress.XtraReports.UI.XtraReport {
        public CategoriesReport() {
            InitializeComponent();
            pbPicture.DataBindings.Add("Image", null, "Picture");
        }

        private void CategoriesReport_DataSourceDemanded(object sender, EventArgs e) {
            this.DataSource = Category.GetCategories();
        }

    }
}
